org.pipoypipagames.cowsrevenge.json

```diff
@@ -0,0 +1,29 @@
+{
+  "app-id": "org.pipoypipagames.cowsrevenge",
+  "branch": "stable",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "cowsrevenge",
+  "finish-args": [
+     "--socket=x11",
+     "--socket=pulseaudio",
+     "--filesystem=home"
+  ],
+  "build-options" : {
+    "split" : false,
+    "no-debuginfo" : true
+  },
+  "modules": [
+    {
+      "name": "cowsrevenge",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/Dariasteam/Cows-Revenge-Flatpak",
```

Comment:
Which architectures does this work on? It looks like there's just an x86_64 binary at this domain. You need to use a flathub.json file to prevent it being offered on other systems where it won't work.

Example here: https://github.com/flathub/us.zoom.Zoom/commit/c669b61fa49c032b8c2220f8622af33effaec390

Reply:
Thank you, gonna change it now

Reply:
There is some list with all architectures codenames? Can't find anything in flatpak documentation

Reply:
"aarch64", "arm" and "i386" are the ones you want.